Tests/OpenCombineTests/PublisherTests/ReplaceNilTests.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ReplaceNilTests.swift
+//  
+//
+//  Created by Joseph Spadafora on 7/4/19.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class ReplaceNilTests: XCTestCase {
+    static let allTests = [
+        ("testReturnsMapPublisher", testReturnsMapPublisher),
+        ("testReplacesNilElement", testReplacesNilElement),
+        ("testExistingElementIsPreserved", testExistingElementIsPreserved)
+    ]
+
+    func testReturnsMapPublisher() {
+        // Given
+        let nilPublisher = PassthroughSubject<Int?, Never>()
+
+        // When
+        let replaceNil = nilPublisher.replaceNil(with: 42)
+
+        // Then
+        let mirror = Mirror(reflecting: replaceNil)
+        let expected = "Mirror for Map<PassthroughSubject<Optional<Int>, Never>, Int>"
+        XCTAssertEqual(mirror.description, expected)
+    }
```

Comment:
That's true, but when we are running in compatibility mode if there is an update that changes how replace nil is implemented to use something different with map, this will catch it.  I don't think there's any harm in keeping it.

Reply:
In this case, you can just explicitly annotate types, so it won't compile if the implementation is ever changed.

Reply:
@broadwaylamb I've removed the test